pkg/kubectl/cmd/util/helpers_test.go

```diff
@@ -174,6 +174,144 @@ func TestMerge(t *testing.T) {
 				},
 			},
 		},
+		{
+			kind: "Pod",
+			obj: &api.Pod{
+				ObjectMeta: api.ObjectMeta{
+					Name: "foo",
+				},
+			},
+			fragment: `{ "apiVersion": "v1" }`,
+			expected: &api.Pod{
+				ObjectMeta: api.ObjectMeta{
+					Name: "foo",
+				},
+				Spec: api.PodSpec{
+					RestartPolicy: api.RestartPolicyAlways,
+					DNSPolicy:     api.DNSClusterFirst,
+				},
+			},
+		},
+		/* TODO: uncomment this test once Merge is updated to use
+		strategic-merge-patch. See #844.
```

Comment:
Issue number has to be wrong. 

Instead of inserting comment-out code, how about just putting a todo above the corresponding test case to duplicate or convert to v1?